portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,9 +70,33 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String content = request.getParameter("comment");
     String name = request.getParameter("name");
-    if (content.length() != 0) {
-      Comment comment = name.length() == 0 ? new Comment(content) : new Comment(name, content);
-      commentsList.add(comment);
+    String stringifiedKey = request.getParameter("comment-key");
+
+    /* 
+      First block executes when a new comment is posted.
```

Comment:
Use a consistent commenting style: https://google.github.io/styleguide/javaguide.html#s4.8.6-comments 
either inline // or 
/**
 * First...
 * Second block....
 */

Reply:
oh that formatting totally didn't come out right, but you get the idea....